packages/template-retail-react-app/app/components/login/index.jsx

```diff
@@ -71,6 +75,8 @@ const LoginForm = ({submitForm, clickForgotPassword = noop, clickCreateAccount =
                             />
                         </Button>
 
+                        <SocialLogin idps={idps} />
+
```

Comment:
Would we need some conditional above this that if idps we show the "Or Login From" text above the IDPs

Reply:
In your PR, you should now pass the SocialLogin into your component